requirements.txt

```
@@ -1,6 +1,6 @@
 numpy==1.22.0
 pandas==0.18.1
-scipy==0.17.1
+scipy==1.10.0
```

```suggestion
scipy==1.10.0
```
- Now, scipy is updated to version 1.10.0. Remember to use the correct package versions as per your project requirements.